web-test-runner.utils.js

```diff
@@ -47,6 +46,29 @@ export const chromiumWithMemoryTooling = playwrightLauncher({
     },
 });
 
+export const chromiumWithMemoryToolingCI = playwrightLauncher({
+    product: 'chromium',
+    concurrency: 1,
```

Comment:
@Rajdeepc while you're working through the new VRT failures here, can you test whether the separation of unit and memory tests allows us to leverage concurrency to get the memory test into the same general time scale as the unit/list/preview tasks? If we could get this one under 5 minutes as well, that would be great!

Reply:
<img width="468" alt="Screenshot 2024-05-10 at 4 27 52 PM" src="https://github.com/adobe/spectrum-web-components/assets/8790510/d7a9a8bc-207c-4fbc-ac61-9d512c56219f">